meson_options.txt

```diff
@@ -4,6 +4,7 @@ option('cli', type: 'feature', value: 'auto', description: 'Build CLI programs (
 option('static_runtime', type: 'boolean', value: false, description: 'Set to true when building a static version of Rizin')
 option('local', type: 'feature', value: 'auto', description: 'Adds support for local/side-by-side installation (sets rpath if needed)')
 option('blob', type: 'boolean', value: false, description: 'Compile just one binary which dispatch to the right handlers based on the name used to call it')
+option('no_subprojects_check', type: 'boolean', value: false, description: 'Do not check if git subprojects are up-to-date. Useful when developing on a different subproject version')
```

Comment:
maybe use affermative option instead of negative one. `option(sub-projects_check, value: true, ..)`